### PR TITLE
Category opens in vue.js frontend

### PR DESCRIPTION
**Pull request by Gigiman**

### CODE REVIEW

File: front/client/App.vue [hunk 1]
```diff
@@ -3,7 +3,9 @@
     <toolbar></toolbar>
     <v-content>
       <v-container fluid>
-        <router-view></router-view>
+        <keep-alive exclude="article-page">
```
Comment (avele) [inline front/client/App.vue:6]: Why do you need keep-alive for almost all the app, except article-page?

File: front/client/components/Article.vue [hunk 2]
```diff
@@ -8,12 +8,12 @@
           v-for="(value, key) in getCategory"
           :key="key"
         >
-          <a
+          <router-link
             class="article-top-link"
-            href="https://guide.aelve.com/haskell/lenses-sth6l9jl"
+            :to="`${pathname()}`"
```
Comment (avele) [inline front/client/components/Article.vue]: Why use function but not getter? Also bad name, pathname to where?

File: front/client/components/Article.vue [hunk 3]
```diff
@@ -78,8 +81,11 @@ import AddItemDialog from 'client/components/AddItemDialog.vue'
 export default class ArticleItem extends Vue {
   isDialogOpen: boolean = false
 
-  async asyncData () {
-    await this.$store.dispatch('categoryItem/loadCategoryItem')
+  async asyncData() {
+    const rawUrl = this.$route.params.category
```
Comment (avele) [inline front/client/components/Article.vue:87]: You dont need to access router params, just pass it as props. https://router.vuejs.org/guide/essentials/passing-props.html

File: front/client/App.vue (hunk 1 above)
Comment (Gigiman) [reply front/client/App.vue:6]: there is an issue about cache whick I was not able to solve even with keep-alive. Please take a look

**Pull request merged**